.github/workflows/workflow-cleanup.yml

```
@@ -0,0 +1,35 @@
+name: Cleanup old and orphaned workflow runs
+
+on: [pull_request] # TODO: remove before merge to run on default branch
+  #schedule:
+    #- cron: '0 12 * * *' # runs at 12:00 UTC everyday
```

thought this might save you a trip.  feel free to do it yourself if want a cleaner blame though
```suggestion
  schedule:
    - cron: '0 12 * * *' # runs at 12:00 UTC everyday
```